### ISSUE TITLE
npm run coverage results in one error

### ISSUE THREAD

**Issue by radiocontrolled** [bbc/a11y-tests-web#17]
## Summary

`npm run coverage` results in a failing test on branch `master`

## Expected Behaviour

Tests should pass 

## Current Behaviour
output from `npm run coverage` is:

```
  1) bbcA11y build() Paths and baseUrl and visit and options outputs the basic config for the paths, with the defined baseUrl, and with the visit function:
     AssertError: expected writeFileSync to be called with arguments
a11y.js


    page(
      "http://base.url/path/1",
      {
        visit: function (){/* Do something */cov_17hr8di7tb.f[0]++;},
        "some":"option"
      }
    )

    page(
      "http://base.url/path/2",
      {
        visit: function (){/* Do something */cov_17hr8di7tb.f[0]++;},
        "some":"option"
      }
    )


   match()
      at Object.fail (node_modules/sinon/lib/sinon/assert.js:96:21)
      at failAssertion (node_modules/sinon/lib/sinon/assert.js:55:16)
      at Object.assert.(anonymous function) [as calledWith] (node_modules/sinon/lib/sinon/assert.js:80:13)
      at Context.it (test/lib/bbcA11y.js:175:24)
```

## Your Environment

* version "5.1.0"`

**Comment by andymsuk** [comment]
Good spot, @radiocontrolled. I'll look into it.

**Comment by andymsuk** [comment]
Hi @radiocontrolled.

Thanks to @mattgreenham, this issue is now resolved and released under v5.2.1.
